src/filesystem/impls/filer/lib/CSSRewriter.js

```diff
@@ -28,6 +28,7 @@ define(function (require, exports, module) {
         function aggregate(content, callback) {
             var urls = [];
             var urlRegex = new RegExp('url\\([\\\'\\"]?([^\\\'\\"\\)]+)[\\\'\\"]?\\)', 'g');
+            var urlQuotesRegex = new RegExp(/[\\\'\\"]([^\\\'\\"\\)]+)[\\\'\\"]/, 'g');
```

Comment:
Sorry. I was caught up in some work in the past 2 weeks and I'm a bit sick since a couple of days 😅. I'll get onto it asap.